themes/vue/source/css/_settings.styl

```diff
@@ -23,7 +23,7 @@ $header-inner-height = 41px
 $heading-padding-vertical = 10px
 $header-height = $header-inner-height + $heading-padding-vertical * 2
 $mobile-header-height = 40px
-$heading-link-padding-top = $content-padding-top
-$mobile-heading-link-padding-top = $content-padding-top
+$heading-link-padding-top = $header-height + $content-padding-top
```

Comment:
For the former mistake, I advice to add some comment here and `function makeHeaderClickable()` (whatever it will become finally).